hexane/atomistic/Extract_Energies.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+if [ -d energies ]; then
+  rm -r energies
+fi
```

Comment:
If you are going to delete it anyhow, a simple `rm -rf energies` would to the same.